gms/versioned_value.hh

```diff
@@ -41,29 +41,27 @@ class versioned_value {
 public:
     // this must be a char that cannot be present in any token
     static constexpr char DELIMITER = ',';
-    static constexpr const char DELIMITER_STR[] = { DELIMITER, 0 };
+    static constexpr std::string_view DELIMITER_STR{&DELIMITER, 1};
```

Comment:
It looks like we can get rid of `DELIMITER` altogether and then the definition of `DELIMITER_STR` would be just `","`, more readable.

Reply:
done